.drone-v1.yml

```diff
@@ -34,16 +34,14 @@ steps:
   image: quay.io/ukhomeofficedigital/trivyscanner:master
   commands:
   - trivy --version
-  - trivy --ignore-unfixed --exit-code 0 --no-progress quay.io/ukhomeofficedigital/dq-kube-monitor-dashboard:${DRONE_COMMIT_SHA}
+  - trivy image --ignore-unfixed --exit-code 1 --no-progress quay.io/ukhomeofficedigital/dq-kube-monitor-dashboard:${DRONE_COMMIT_SHA}
```

Comment:
Why change from an error code of 0 ("no error") to 1 ("an error")?

Reply:
In order fail the pipeline if a security vulnerability is discovered